app/components/Form/ProjectContactForm.tsx

```diff
@@ -100,13 +100,6 @@ const ProjectContactForm: React.FC<Props> = (props) => {
     props.query
   );
 
-  const lastEditedDate = useMemo(() => {
-    const mostRecentUpdate = projectContactFormChanges.edges
-      .map((e) => e.node.updatedAt)
-      .sort((a, b) => Date.parse(b) - Date.parse(a))[0];
-    return new Date(mostRecentUpdate);
-  }, [projectContactFormChanges]);
-
```

Comment:
now that this is removed, your linter should show some warnings on the `updatedAt` field in your relay fragment above, telling you it's not used. It can be removed. Maybe do a search for `updatedAt` in our tests too, a bunch of them might need to be removed from the mock query payloads